project/frontend/src/components/ResultsMap.js

```diff
@@ -3,39 +3,54 @@ import React, { useState } from 'react';
 
 const mapStyle = {
   border: 'thin solid black',
-  height: '500px',
-  width: '500px',
+  height: '100%',
+  width: '100%',
 };
 
 function MapContainer(props) {
-  const [activeMarker, changeMarker] = useState({});
-  const [showInfoWindow, changeInfoVisibility] = useState(false);
-
-  const coords = { lat: 40.837, lng: -73.865 };
+  const [activeMarker, setActiveMarker] = useState({});
+  const [showInfoWindow, setShowInfoWindow] = useState(false);
+  const restaurants = props.restaurants;
+  const userCenter = props.userLocation;
 
   const onMarkerClick = (props, marker) => {
-    changeMarker(marker);
-    changeInfoVisibility(true);
+    setActiveMarker(marker);
+    setShowInfoWindow(true);
   };
 
-  const onClose = (props) => {
+  const onClose = () => {
     if (showInfoWindow) {
-      changeInfoVisibility(false);
+      setShowInfoWindow(false);
+    }
+  };
+
+  const createMarkers = () => {
+    if (restaurants.length >= 1) {
+      let markers = [];
+      const numOfMarkers = Math.min(restaurants.length, 4);
+      for (let i = 0; i < numOfMarkers; i++) {
+        const coords = restaurants[i].key.latLngCoords;
+        markers.push(
+          <Marker
+            onClick={onMarkerClick}
+            position={coords}
+            name={'Your #' + toString(i + 1) + ' Match'}
+            aria-label={'Your #' + toString(i + 1) + ' Match'}
+          />
+        );
+      }
+      return markers;
     }
```

Comment:
nit: prefer reducing nesting by returning early when possible 
here you can do: 
```
if (restaurants.length == 0) {
    return;
}
let markers = []'
/..... rest of the stuff 
```

Reply:
To be even more concise, rather than `if (restaurants.length == 0) {` you can just use truthy/falsy and say `if (!restaurants) {`